build/extensions/extensions.js

```diff
@@ -9,8 +9,8 @@
 import { createPlugin } from "../../web/client/utils/PluginsUtils";
 
 export default {
-    Extension: createPlugin('Extension', {
+    Extension: createPlugin('SampleExtension', {
```

Comment:
Also this one should change name ?
```suggestion
    SampleExtension: createPlugin('SampleExtension', {
```